### PR TITLE
Rating

### PR DESCRIPTION
**Pull request by c0m1t**

### CODE REVIEW

File: src/shared/rating/Rating.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  inputLabel: string;
+  initialValue?: string;
+}
+
+type Props = FCProps<OwnProps>;
+
+export function Rating(props: Props) {
+  const { inputLabel, initialValue } = props;
+
+  const options = getOptionsFromDictionary(evaluationDictionary);
```
Comment (smmoosavi) [inline src/shared/rating/Rating.tsx]: useMemo

File: src/globalTypes.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { i18n } from '@lingui/core';
```
Comment (sijav) [inline src/globalTypes.ts]: Create a folder named global-types have a file named evaluation.ts put these enum and evaluation dictionary in it, create a types.d.ts index the folder and re export these

File: src/shared/rating/utils.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { Option } from 'src/shared/forminator/types';
```
Comment (sijav) [inline src/shared/rating/utils.ts]: It's better to be on src/shared/utils/getOptionsFromDictionary.ts and have it imported from it

File: src/shared/rating/Rating.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import { i18n } from '@lingui/core';
+import { Box } from '@material-ui/core';
+import { action } from '@storybook/addon-actions';
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { Evaluation } from 'src/globalTypes';
+import { Forminator, SubmitButton } from 'src/shared/forminator';
+import { Rating } from 'src/shared/rating/Rating';
```
Comment (sijav) [inline src/shared/rating/Rating.stories.tsx]: import from ./Rating

Comment (c0m1t) [reply src/shared/rating/Rating.stories.tsx]: they in the same scope

File: src/globalTypes/index.ts [hunk 5]
```diff
@@ -0,0 +1 @@
+export { Evaluation, evaluationDictionary } from './Evaluation';
```
Comment (sijav) [inline src/globalTypes/index.ts]: Change folder name to global-types

File: src/shared/rating/Rating.stories.tsx [hunk 6]
```diff
@@ -3,10 +3,10 @@ import { Box } from '@material-ui/core';
 import { action } from '@storybook/addon-actions';
 import { storiesOf } from '@storybook/react';
 import React from 'react';
-import { Evaluation } from 'src/globalTypes';
+import { Evaluation } from 'src/globalTypes/Evaluation';
```
Comment (sijav) [inline src/shared/rating/Rating.stories.tsx]: import from src/global-types

File: src/locales/en/messages.po [hunk 7]
```diff
@@ -25,29 +25,53 @@ msgstr "Cancel"
 msgid "Criteria"
 msgstr "Criteria"
 
+#: src/globalTypes.ts:11
+msgid "EXCEEDS_EXPECTATIONS"
+msgstr "EXCEEDS_EXPECTATIONS"
```
Comment (sijav) [inline src/locales/en/messages.po]: Tranlate these

File: src/global-types/Evaluation.ts [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+export enum Evaluation {
+  'EXCEEDS_EXPECTATIONS' = 'EXCEEDS_EXPECTATIONS',
+  'MEETS_EXPECTATIONS' = 'MEETS_EXPECTATIONS',
+  'NEEDS_IMPROVEMENT' = 'NEEDS_IMPROVEMENT',
+  'STRONGLY_EXCEEDS_EXPECTATIONS' = 'STRONGLY_EXCEEDS_EXPECTATIONS',
+  'SUPERB' = 'SUPERB',
+}
+
+export const evaluationDictionary: Record<Evaluation, string> = {
+  [Evaluation.EXCEEDS_EXPECTATIONS]: i18n._('EXCEEDS_EXPECTATIONS'),
```
Comment (smmoosavi) [inline src/global-types/Evaluation.ts]: i18n._('Exceeds Expectations')

**Pull request merged**